astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -1,20 +1,20 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-import itertools
 import inspect
+import itertools
 
 import numpy as np
 import numpy.lib.recfunctions as rfn
-from numpy.testing import assert_array_equal
-
 import pytest
+from numpy.testing import assert_array_equal
 
 from astropy import units as u
-from astropy.units.quantity_helper.function_helpers import (
-    ARRAY_FUNCTION_ENABLED, SUBCLASS_SAFE_FUNCTIONS, UNSUPPORTED_FUNCTIONS,
-    FUNCTION_HELPERS, DISPATCHED_FUNCTIONS, IGNORED_FUNCTIONS, TBD_FUNCTIONS)
+from astropy.units.quantity_helper.function_helpers import (ARRAY_FUNCTION_ENABLED,
```

Comment:
Seems worse! what's wrong with the open parenthesis?

Reply:
We could switch astropy to 5 - Hanging Grid Grouped in 
``https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html``

This would make small changes to cosmology and modeling. I'm fine either way.
@WilliamJamieson, what do you think about changing the multi-line arrangement?

Reply:
Just testing 4. It's in a separate commit, so it can easily be rolled back.